packages/location-management/src/components/LocationTree/index.tsx

```diff
@@ -26,6 +26,7 @@ const Tree: React.FC<TreeProp> = (props: TreeProp) => {
 
   const [expandedKeys, setExpandedKeys] = useState<React.Key[]>([]);
   const [searchValue, setSearchValue] = useState<string>('');
+  const [selectedKey, setSelectedKey] = useState<React.Key[]>([]);
```

Comment:
I am a bit skeptical that this fully resolves the issue as linked, specifically will the state not reset during a page refresh, @HaseemUlHaq have you tested this on the browser?

Reply:
@p-netm On reloading the page, we dont need to save the tree state.
UX was also discussed with @dubdabasoduba and team.
On reload, tree state will get back to its default state. It will only persist when pushing from one page to another.

Reply:
by 

> It will only persist when pushing from one page to another.

In order to understand this, How would I be able to check this, what are the steps and observations that I should make?

Reply:
@p-netm 
- Select any location from location tree,
- Go to create new location,
- Click on cancel button
It will redirect back to location table page where tree state should persist.